requirements-git.txt

```diff
@@ -1,2 +1,3 @@
 git+https://github.com/firedrakeproject/fiat.git#egg=fenics-fiat
 git+https://github.com/firedrakeproject/tsfc.git#egg=tsfc
+git+https://github.com/FEniCS/ufl.git#egg=fenics-ufl
```

Comment:
Note for the meeting where we will hopefully merge all this later, is this pointing to the correct branch, or should it be firedrake UFL?

Reply:
Correct fork/mirror rather

Reply:
i.e.
```suggestion
git+https://github.com/firedrakeproject/ufl.git@dolci/merge_upstream_2
```